### ISSUE TITLE
Change redirect from CoP Readme to CoP page

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#5329]
### Overview
We need to change https://www.hackforla.org/cop to redirect to the _Communities of Practice_ (CoP) page [^1] so that it redirects to the expected page on the website.

### Action Items
- [x] In redirections/cop.md [^2], 
  - [x] Change
    ```
    # https://www.hackforla.org/cop redirects to -> https://github.com/hackforla/communities-of-practice/blob/main/README.md
    ```
    to 
    ```
    # https://www.hackforla.org/cop redirects to https://www.hackforla.org/communities-of-practice
    ```
  - [x] Change
    ```
    redirect_to: https://github.com/hackforla/communities-of-practice/blob/main/README.md
    ```
    to
    ```
    redirect_to: /communities-of-practice/
    ```
- [x] Using Docker, check https://www.hackforla.org/cop will redirect to _Communities of Practice_ (CoP) page [^1]

### Resources/Instructions
[^1]: [_Communities of Practice_ (CoP) page](https://www.hackforla.org/communities-of-practice)
[^2]: [redirections/cop.md](https://github.com/hackforla/website/blob/gh-pages/redirections/cop.md?plain=1)

**Comment by JessicaLucindaCheng** [comment]
@hackforla/website-pm I noticed https://www.hackforla.org/cop redirects to https://github.com/hackforla/communities-of-practice/blob/main/README.md. Should it redirect to [_Communities of Practice_ (CoP) page](https://www.hackforla.org/communities-of-practice) instead?
- If yes, then this issue is ready to be prioritized.
- If no, then please add a comment stating the current redirect is desired and close this issue.

Thank you. :smile:

**Comment by vinfinity7** [comment]
Hi @JessicaLucindaCheng!  I am new to the HackerLa platform, so  I just gotta solve the issue and submit PR or is there any registration required before this? Thank You!

**Comment by JessicaLucindaCheng** [comment]
@vinfinity7 Before working on any issues, you need to be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.

**Comment by robertnjenga** [comment]
Availability: 10am - 2pm 
ETA:  9/26/2023

**Comment by JessicaLucindaCheng** [comment]
- The developer did everything correctly and as asked in this issue. 

- Since I wrote the issue, it is my fault it is redirecting to the "Uh Oh!" 404 page instead of the [_Communities of Practice_ (CoP) page](https://www.hackforla.org/communities-of-practice). Sorry :disappointed:. I did test the redirect using Docker to check my changes, which worked as expected, but the live site did something different. To prevent this in the future, I will make sure to double-check any redirects I write in an issue work on a live website in my forked repo and not just with Docker locally on my computer.

- I wrote this issue to fix the problem: #5622
